bundled/kokkos-4.5.01/core/src/View/Kokkos_BasicView.hpp

```diff
@@ -523,7 +523,7 @@ class BasicView {
 
   // Some weird unexplained issue in compiling the SFINAE version with CUDA/MSVC
   // So we just use post factor check here with static_assert
-#if defined(KOKKOS_ENABLE_CUDA) && defined(_WIN32)
+#if defined(KOKKOS_ENABLE_CUDA) || defined(_WIN32)
```

Comment:
```suggestion
  // Some weird unexplained issue in compiling the SFINAE version with MSVC
  // So we just use post factor check here with static_assert
#if defined(_WIN32)
```
is what upstream `Kokkos` is doing.

Reply:
Yes, I saw that later too. I think they're functionally equivalent given that we only ever use the bundled version without CUDA, and the use of the `||` operator then just takes the second branch. I can change it if you want, though I don't think it's important.

Reply:
Yeah, I preferred if we just mirror `upstream` to avoid confusion why we do somethign else here.